### DISCUSSION TITLE
Custom Join/Leave Message

### DISCUSSION THREAD

**Discussion by byjdw** [Zrips/CMI#6022] [Q&A]
Hey! I was wondering where I could find the setting to actually change the join/leave message, since it doesn't seem to be in the config.yml, all you can do is enable it from there.

**Comment by viasnake** [comment]
It can be changed in Translations in CMILib.
As of version 1.0.3.5, it is present in lines 199-201.

```
  # This line can have extra variables: [totalUsers] [onlinePlayers]
  FirstJoin: '&eWelcome &6[playerDisplayName] &eto our server!'
  LogoutCustom: ' &6[playerDisplayName] &eleft the game'
  LoginCustom: ' &6[playerDisplayName] &ejoined the game'
```

**Comment by mrfloris** [comment]
I hope this helps
https://github.com/mrfdev/CMI/blob/master/Resources/FAQ/cmi-custom-joinleave.md
